docs_src/examples/app-service-examples/camera-management/Ch-CameraManagement.md

```diff
@@ -0,0 +1,404 @@
+    ```
+
+1. Build the docker image
+    ```bash
+    make docker
+    ```
+
+1. Start the docker compose services in the background for both EVAM and Camera Management App
+    ```bash
+    docker compose up -d
+    ```
+
+!!! note
+    If you would like to view the logs for these services, you can use `docker compose logs -f`. To stop the services, use `docker compose down`.
+
+!!! note
+    The port for EVAM result streams has been changed from 8554 to 8555 to avoid conflicts with the device-usb-camera service.
+
+## Using the App
+
+1. Visit http://localhost:59750 to access the app.
+
+![homepage](./images/homepage-demo-app-1.png)  
+<p align="left">
+<i>Figure 1: Homepage for the Camera Management app</i>
+</p>
+
+### Camera Position
+
+You can control the position of supported cameras using ptz commands.
+
+![camera-position](./images/camera-position.png)  
+
+1. Use the arrows to control the direction of the camera movement.
+1. Use the magnifying glass icons to control the camera zoom.
+
+### Start an Edge Video Analytics Pipeline
+
+This section outlines how to start an analytics pipeline for inferencing on a specific camera stream.
+
+![camera](./images/camera.png)  
+
+1. Select a camera out of the drop down list of connected cameras.  
+   ![select-camera](./images/select-camera.png)  
+
+1. Select a video stream out of the drop down list of connected cameras.  
+   ![select-profile](./images/select-profile.png)  
+
+1. Select a analytics pipeline out of the drop down list of connected cameras.  
+   ![select-pipeline](./images/select-pipeline.png)  
+
+1. Click the `Start Pipeline` button.
+
+
+### Running Pipelines
+
+Once the pipeline is running, you can view the pipeline and its status.
+
+![default pipelines state](./images/multiple-pipelines-default.png)  
+
+1. Expand a pipeline to see its status. This includes important information such as elapsed time, latency, frames per second, and elapsed time.  
+   ![select-camera](./images/running-pipelines.png)  
+
+1. In the terminal where you started the app, once the pipeline is started, this log message will pop up.
+    ```bash
+    level=INFO ts=2022-07-11T22:26:11.581149638Z app=app-camera-management source=evam.go:115 msg="View inference results at 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'"
+    ```
+
+1. Use the URI from the log to view the camera footage with analytics overlayed.
+    ```bash
+    ffplay 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'
+    ```
+
+   Example Output:  
+   ![example analytics](./images/example-analytics.png)  
+
+1. Press the red square stop button to shut down the pipeline.
+
+
+### API Log
+
+The API log shows the status of the 5 most recent calls and commands that the management has made. This includes important information from the responses, including camera information or error messages.
+
+![API Logs](./images/api-log.png)  
+
+1. Expand a log item to see the response
```

Comment:
I think this numbering is not required and does not look good while following the docs.